external-crates/move/crates/move-vm-types/src/values/values_impl.rs

```diff
@@ -1205,10 +1205,22 @@ impl SignerRef {
  *
  **************************************************************************************/
 impl Locals {
-    pub fn new(n: usize) -> Self {
-        Self(Rc::new(RefCell::new(
-            iter::repeat_with(|| ValueImpl::Invalid).take(n).collect(),
-        )))
+    // Legacy -- will be deprecated.
+    pub fn new(size: usize) -> Self {
+        Locals::new_from(vec![], size)
+    }
```

Comment:
Are there calls to this still?

Reply:
There are in the test suites for the cut versions.